### PR TITLE
Use Azure front door endpoint to call OP API

### PR DESCRIPTION
**Pull request by 928PJY**
[AB#380821](https://dev.azure.com/ceapex/d3d54af3-265a-4f18-95f6-9a46397ca583/_workitems/edit/380821)

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/docfx/pull/7084)

### CODE REVIEW

File: src/docfx/config/ops/OpsAccessor.cs [hunk 1]
```diff
@@ -32,12 +32,11 @@ internal class OpsAccessor : ILearnServiceAccessor
 
         public static readonly DocsEnvironment DocsEnvironment = GetDocsEnvironment();
 
-        // TODO: use Azure front door endpoint when it is stable
         private static readonly string s_docsProdServiceEndpoint =
-            Environment.GetEnvironmentVariable("DOCS_PROD_SERVICE_ENDPOINT") ?? "https://op-build-prod.azurewebsites.net";
+            Environment.GetEnvironmentVariable("DOCS_PROD_SERVICE_ENDPOINT") ?? "https://buildapi.docs.microsoft.com";
```
Comment (yufeih) [inline src/docfx/config/ops/OpsAccessor.cs:36]: Do you still need `DOCS_PROD_SERVICE_ENDPOINT`?

Comment (928PJY) [reply src/docfx/config/ops/OpsAccessor.cs:36]: This is just a default value, Docs.Build will pass this environment variable, maybe we can still keep this override mechanism?

Comment (yufeih) [reply src/docfx/config/ops/OpsAccessor.cs:36]: If there is no value in this override, I'd like to remove it for simplicity

**Comment by mingwli** [comment]
@928PJY Seems our token in keyvault docfx is invalid. The response states [401](https://ceapex.visualstudio.com/d3d54af3-265a-4f18-95f6-9a46397ca583/_apis/build/builds/337876/logs/81), not [403](https://ceapex.visualstudio.com/d3d54af3-265a-4f18-95f6-9a46397ca583/_apis/build/builds/337798/logs/91)

**Pull request merged**